cmd/livepeer/livepeer.go

```diff
@@ -133,9 +133,9 @@ func main() {
 	ethKeystorePath := flag.String("ethKeystorePath", "", "Path for the Eth Key")
 	ethOrchAddr := flag.String("ethOrchAddr", "", "ETH address of an on-chain registered orchestrator")
 	ethUrl := flag.String("ethUrl", "", "Ethereum node JSON-RPC URL")
-	ethController := flag.String("ethController", "", "Protocol smart contract address")
-	gasLimit := flag.Int("gasLimit", 0, "Gas limit for ETH transactions")
```

Comment:
Let's follow the same deprecation procedure that we're using for `-gasPrice` for `-gasLimit` as well.

Reply:
decided to keep around `-gasLimit` and not deprecate it for now even though I'm not sure how useful the flag is.